go.mod

```diff
@@ -192,6 +193,7 @@ require (
 
 replace (
 	github.com/apcera/gssapi => github.com/openshift/gssapi v0.0.0-20161010215902-5fb4217df13b
+	golang.org/x/crypto/openpgp => github.com/ProtonMail/go-crypto/openpgp v0.0.0-20220623141421-5afb4c282135
```

Comment:
This PR directly imports `github.com/ProtonMail/go-crypto/openpgp`, is the replace required?